content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -38,7 +38,8 @@ You can run the example with this command:
 ```shell
 kubectl apply -f https://kubernetes.io/examples/controllers/job.yaml
 ```
-```
+The output is similar to this:
+```shell
```

Comment:
This line is incorrect @ydFu
It should be: <tt>\`\`\`</tt>

Reply:
Thanks for your guidance.